config/locales/server.en.yml

```diff
@@ -2387,6 +2387,7 @@ en:
       email_polling_disabled: "You must enable either manual or POP3 polling before enabling reply by email."
       user_locale_not_enabled: "You must first enable 'allow user locale' before enabling this setting."
       invalid_regex: "Regex is invalid or not allowed."
+      invalid_regex_with_message: "Regex %{regex} is invalid: %{message}"
```

Comment:
How about a more sentence-like error message like this?

`The regular expression '%{regex}' contains the following error: %{message}`

Reply:
I do not have an opinion about either option. I noticed we are using both styles in our site settings errors. I prefer a shorter message because `%{message}` will expand too.